docs/chapters/sdk/template-service.rst

```diff
@@ -0,0 +1,19 @@
+Template Service
+================
+
+A template service serves as an example CMake_ project for service developers that should be used as a starting point while creating platform services. To acquire the template service source code the user needs to fetch the code from github.
```

Comment:
@breakreturn thanks for the comment. I will move the documentation from the internal wiki to here and possible expand it if needed.